.pre-commit-config.yaml

```diff
@@ -15,10 +15,6 @@ repos:
       - id: check-yaml
       - id: check-shebang-scripts-are-executable
       - id: check-case-conflict
-  - repo: https://github.com/lovesegfault/beautysh
-    rev: v6.2.1
-    hooks:
-      - id: beautysh
```

Comment:
If you are removing beautysh, add the replacement hooks for it? I won't accept just removing something without any replacement.

Reply:
Done